src/cloud-cli/meltano/cloud/README.md

```diff
@@ -0,0 +1,85 @@
+# Meltano Cloud CLI
+
+`meltano-cloud` is a standalone CLI tool for interacting with the Meltano Cloud API.
+
+## Configuration
+
+`meltano-cloud` can be configured via
+`meltano-cloud config set <setting name> <setting value>`.
+
+### Configuration File
+
+Meltano Cloud configuration settings and credentials are stored in a JSON file.
+`meltano-cloud` searches for this configuration file using [platformdirs](https://github.com/platformdirs/platformdirs).
+
+If a configuration file is found but the configuration it contains
+is invalid, `meltano-cloud` will raise an error rather than continuing its search.
+
+### Settings
+
+The following settings are configurable:
+
+#### `auth_callback_port`
+
+As part of authentication, `meltano-cloud` runs a lightweight, short-lived HTTP
+server on `localhost` to handle authentication callbacks. By default, this
+server runs on port 8080 but can be configured to another port using this setting.
```

Comment:
Is this out of date? I thought port 9999 was being used? This port number also appears in a couple other spots.

Reply:
yep, that is out of date. Updated. 

The only other place I could find it was in unit tests, which is fine. It can use any port locally but cognito will throw an error if you try to redirect to anything besides 9999 on localhost.